salt/modules/gnomedesktop.py

```diff
@@ -62,7 +62,7 @@ def _get(self):
             log.info('User does not exist')
             return False
 
-        cmd = 'dbus-launch --exit-with-session gsettings get {0} {1}'.format(self.SCHEMA, self.KEY)
+        cmd = 'dbus-run-session -- gsettings get {0} {1}'.format(self.SCHEMA, self.KEY)
```

Comment:
can you add a check to make sure that this command is actually there?

Since it looks like it is easily interchangable, it would be nice to just make sure that older setups that only have dbus-launch can still work.

Thanks,
Daniel

Reply:
Done.